src/main/java/eu/miaplatform/service/environment/EnvConfiguration.java

```diff
@@ -57,7 +57,7 @@ public static void parseEnvironment() throws InvalidEnvConfigurationException {
 
     private static String getRequired(String envKey) throws InvalidEnvConfigurationException {
         String value = System.getenv(envKey);
-        if (value != null && !value.isEmpty()) {
```

Comment:
Can we add a test verifying this code branches? The fact that we didn't notice this error is a signal of test lack.